src/Containers/SignUpForm/SignUpForm.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty scss file? Do we need to add this?

Reply:
Good catch! I forget to delete it after I replaced styles to welcome page styles.